axelrod/tests/strategies/test_better_and_better.py

```diff
@@ -23,5 +23,14 @@ class TestBetterAndBetter(TestPlayer):
     def test_strategy(self):
         """Tests that the strategy gives expected behaviour."""
         self.first_play_test(D, seed=3)  # D is very unlikely
```

Comment:
Similar to this: https://github.com/Axelrod-Python/Axelrod/blob/master/axelrod/tests/strategies/test_averagecopier.py#L25